src/identity-providers/add/DiscoverySettings.tsx

```diff
@@ -33,6 +39,10 @@ const Fields = ({ readOnly }: DiscoverySettingsProps) => {
     control: control,
     name: "config.useJwksUrl",
   });
+  const isPkceEnabled = useWatch({
+    control: control,
```

Comment:
If the key and value are the same you can omit the value.
```suggestion
    control,
```